### ISSUE TITLE
Reports error when using a hidden priority: 0, votes: 0 member

### ISSUE THREAD

**Issue by znewsham** [eladnava/mongomonitor#6]
Replica set configuration on host "..." contains an even number of members - however, the 4th member is non-voting, non-electable and hidden

**Comment by eladnava** [comment]
Hi @znewsham,
Thanks so much for the bug report!

Seems like we should either improve the logic behind the voting majority check or drop it altogether. 

Can you please post your `rs.status()` output, removing any sensitive information, so that we can see if there are any specific flags or JSON properties for your 4th non-voting, non-electable member? We can then check for these properties and exclude such members from the voting majority check if they are non-electable.

**Comment by znewsham** [comment]
@eladnava I ended up doing a bit of an inline fix, but it seemed to work OK

``` 
   var config = yield db.executeDbAdminCommand({ replSetGetConfig: 1 });
     config = config.config;
     var members = rs.members.filter(function(rsMember) {
        const configMember = config.members.filter(function(configMember) {
           return configMember._id == rsMember._id;
        })[0];
        return !configMember || (!configMember.hidden || configMember.votes !== 0 || configMember.priority !== 0);
   });
   if (members.length % 2 === 0) { ...
```

This is slightly overkill - but I wasn't sure whether it was the priority === 0, or votes === 0 which stopped this from being an issue. I'd suggest summing over the # of votes available and checking for odd rather than checking the number of members. 

Unfortunately the information necessary does not seem to be available from `rs.status()`

Incidentally, thanks for this awesome tool!

**Comment by eladnava** [comment]
@znewsham Seems like this info is available from `rs.conf()` which is good enough. Will modify `mongomonitor` to check for this in the next few days. 👍

**Comment by znewsham** [comment]
are you open to PR's?

**Comment by eladnava** [comment]
@znewsham Absolutely! 😄 

The change should be performed in the following section:
https://github.com/eladnava/mongomonitor/blob/55345a1e95b6067fd3234b4dcb300e41fcfcba75/lib/monitor.js#L66-L69

I think replacing this logic with a method `monitor.prototype.verifyElectionMajority`, calling `rs.conf()` inside it, and performing the relevant check there would be splendid 👍 

Let me know if you have any questions.

**Comment by eladnava** [comment]
Closed via #7.
